### ISSUE TITLE
One shot != kill

### ISSUE THREAD

**Issue by markembling** [robhawkes/rawkets#5]
One shot kill is a bit evil. A single stray bullet and you're dead. It would make more sense if it took more shots to kill, but each rocket had either shields (which lose strength when shot and recharge over time), or the concept of damage/structural integrity which fails after a set number of shots.

**Comment by johnpbloch** [comment]
I think 1 shot == kill is just fine. I think a better way to solve your problem is to shorten the range of the rockets.

**Comment by markembling** [comment]
I disagree. If I understood correctly, there was/is going to be an overall aim to the game which doesn't revolve around the concept of killing. If so, being shot every so often (which is very easy to have happen) is going to totally ruin your fun.

Obviously it's not up to me, but I do think that getting killed all the time can get somewhat tedious.

**Comment by robhawkes** [comment]
I'm planning on implementing some sort of shield or health based system in which you need to be shot a few times before dying. I'm also contemplating having that shield or health deplete temporarily every time you fire your weapon as well, this way it becomes dangerous to keep holding down the fire button.

**Comment by markembling** [comment]
That would make more sense after a given time period, i.e. a few moments of holding fire. It also makes sense from a believability point of view as it's quite possible that maintaining shields plus maintaining sustained weapons fire requires more power than the vessel's power source can provide.
